common/src/main/java/com/wynntils/services/statistics/type/StatisticKind.java

```diff
@@ -11,20 +11,23 @@
 import net.minecraft.stats.StatFormatter;
 
 public enum StatisticKind {
+    // region Gameplay
     DAMAGE_DEALT(CustomStatFormatters.FORMATTED_NUMBER, StatisticType.ADVANCED),
     SPELLS_CAST(CustomStatFormatters.FORMATTED_NUMBER, StatisticType.COUNT),
+    // endregion
 
     // region Lootruns
-
+    MYTHICS_FOUND(CustomStatFormatters.FORMATTED_NUMBER, StatisticType.COUNT),
     LOOTRUNS_COMPLETED(CustomStatFormatters.FORMATTED_NUMBER, StatisticType.COUNT),
     LOOTRUNS_FAILED(CustomStatFormatters.FORMATTED_NUMBER, StatisticType.COUNT),
     LOOTRUNS_CHALLENGES_COMPLETED(CustomStatFormatters.FORMATTED_NUMBER, StatisticType.ADVANCED),
+    LOOTRUNS_CHALLENGES_WITHOUT_MYTHIC(CustomStatFormatters.FORMATTED_NUMBER, StatisticType.ADVANCED),
```

Comment:
Maybe rename this to PULLS instead of CHALLENGES as it's storing the dry pull count and they aren't 1:1 due to purple and dark grey beacons

Reply:
I would have to rename the upper one as well, but I can't, since we can't up-fix it. The localizations already say pulls for this reason. Ill change the new one's name, and add a comment for the old one.